server/sonar-server/src/main/java/org/sonar/server/ui/ws/GlobalAction.java

```diff
@@ -122,7 +125,27 @@ private void writeQualifiers(JsonWriter json) {
   }
 
   private void writeVersion(JsonWriter json) {
-    json.prop("version", server.getVersion());
+    String technicalVersion = server.getVersion();
+    String functionalVersion = technicalVersion;
```

Comment:
I'm not sure to understand why are you using 2 variables. Only one should be enough (just replace technicalVersion by server.getVersion() in next line) => it will make the code more clear IMO.